docs/integrations/prometheus.md

```diff
@@ -32,7 +32,8 @@ clusters:
 # We created a namespace called "default" and had set it to retention "48h".
        - namespace: default
          retention: 48h
-         storageMetricsType: unaggregated
+         resolution: 1m
```

Comment:
maybe add a little blurb that this is optional and what it does

Reply:
Hm, this was an oopsy, I didn't mean to make this aggregated hah.